### ISSUE TITLE
results

### ISSUE THREAD

**Issue by 05063112lcs** [TQTQliu/ET-MVSNet#24]
Hello author, I recently studied your code and came to the following conclusions:
1. Your work mainly focuses on feature enhancement and integrating the ET module into the FPN network.
2. Cost body generation, regularization, and depth estimation are not the focus of your work. Can I make improvements in these aspects in the future?

**Comment by TQTQliu** [comment]
Of course, you can improve the performance from the aspects you mentioned.
